-posts/0000-01-02-juhongseo.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
this is difficult to understand the task. it is not really changing but adding a comment to a change proposed